src/main/java/uk/ac/ebi/eva/pipeline/configuration/validation/step/VepInputGeneratorStepParametersValidator.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation.step;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.JobParametersValidator;
+import org.springframework.batch.core.job.CompositeJobParametersValidator;
+import org.springframework.batch.core.job.DefaultJobParametersValidator;
+
+import uk.ac.ebi.eva.pipeline.configuration.JobParametersNames;
+import uk.ac.ebi.eva.pipeline.configuration.validation.ConfigRestartabilityAllowValidator;
+import uk.ac.ebi.eva.pipeline.configuration.validation.DbCollectionsVariantsNameValidator;
+import uk.ac.ebi.eva.pipeline.configuration.validation.DbNameValidator;
+import uk.ac.ebi.eva.pipeline.configuration.validation.InputStudyIdValidator;
+import uk.ac.ebi.eva.pipeline.configuration.validation.InputVcfIdValidator;
+import uk.ac.ebi.eva.pipeline.configuration.validation.OutputDirAnnotationValidator;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Validates the job parameters necessary to execute a {@link uk.ac.ebi.eva.pipeline.jobs.steps.VepInputGeneratorStep}
+ * <p>
+ * The parameters OUTPUT_DIR_ANNOTATION, INPUT_STUDY_ID and INPUT_VCF_ID are used to build the vep input option.
```

Comment:
Please add a reference to where this is done, otherwise it could be a bit confusing.

Also, "VEP" instead of "vep".